neuralpredictors/data/transforms.py

```diff
@@ -298,24 +299,24 @@ def __init__(self, data, stats_source="all", exclude=None):
         itransforms[in_name] = lambda x: x * self._inputs_std + self._inputs_mean
 
         # -- responses
-        transforms["responses"] = lambda x: x * self._response_precision
-        itransforms["responses"] = lambda x: x / self._response_precision
+        transforms[out_name] = lambda x: x * self._response_precision
+        itransforms[out_name] = lambda x: x / self._response_precision
 
-        if "eye_position" in data.data_keys:
-            # -- eye position
-            self._eye_mean = np.array(
-                data.statistics["eye_position"][stats_source]["mean"]
-            )
-            self._eye_std = np.array(
-                data.statistics["eye_position"][stats_source]["std"]
-            )
-            transforms["eye_position"] = lambda x: (x - self._eye_mean) / self._eye_std
-            itransforms["eye_position"] = lambda x: x * self._eye_std + self._eye_mean
 
+        # -- behavior
+        transforms['behavior'] = lambda x: x
+
+        if "pupil_center" in data.data_keys:
```

Comment:
make this work with either "pupil_center" or "eye_position" and not just overwrite. Static datasets use "pupil_center" and movie datasets are using "eye_position"